lib/scss/stanford_person.views.scss

```diff
@@ -17,17 +17,39 @@
   }
 }
 
+.su-person-grid-view,
+.section-grid-view {
+  .main-region {
+    @include grid-media-min(xl) {
+      @include modular-spacing('padding-left', 2);
+    }
+  }
+}
+
 // Grid set up
 .stanford-people-grid {
 
+  // Changing to a Css Grid
+  .flex-container {
+    display: grid;
```

Comment:
Do you need a `row-gap` or `column-gap` here too?

Reply:
Yes I think a column gap. We have margin bottom for between the rows. I'll also added a word break so that it doesn't push the boxes wide.